torch/utils/cpp_extension.py

```diff
@@ -139,12 +152,19 @@ def _define_torch_extension_name(self, extension):
 
 def CppExtension(name, sources, *args, **kwargs):
     '''
-    Create a setuptools.Extension instance for C++.
+    Create a `setuptools.Extension` for C++.
```

Comment:
single backticks are a bit special in RST, and I don't think they will correctly link the the class by default. either use `:class:` before them, or use double backticks